driver/collector/mysql_collector.py

```diff
@@ -213,28 +203,20 @@ def collect_metrics(self) -> Dict[str, Any]:
         else:
             metrics["global"]["engine"]["master_status"] = ""
 
+        metrics["global"]["performance_schema"][
+            "events_statements_summary_by_digest"
+        ] = json.dumps(self._make_list(*self._cmd(self.QUERY_DIGEST_TIME)))
         if float(self._version) >= 8.0:
             # latency histogram
```

Comment:
does this comment still apply?

Reply:
Yes, it's still collecting latency histogram